src/test/java/com/shopify/PrivateAppTest.java

```diff
@@ -0,0 +1,62 @@
+package com.shopify;
```

Comment:
Thanks for testing out generating the authorization header, but is this needed? Can you remove if not? I see that your code does exactly was is described here:
https://shopify.dev/apps/auth/basic-http